storage/table_batch.go

```diff
@@ -131,26 +131,14 @@ func (t *TableBatch) MergeEntity(entity *Entity) {
 // the changesets.
 // As per document https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/performing-entity-group-transactions
 func (t *TableBatch) ExecuteBatch() error {
-
-	// Using `github.com/marstr/guid` is in response to issue #947 (https://github.com/Azure/azure-sdk-for-go/issues/947).
-	id, err := guid.NewGUIDs(guid.CreationStrategyVersion1)
-	if err != nil {
-		return err
-	}
-
-	changesetBoundary := fmt.Sprintf("changeset_%s", id.String())
+	changesetBoundary := fmt.Sprintf("changeset_%s", uuid.NewV1())
```

Comment:
We cannot take this change, because the fundamental error in @satori's code introduced here: https://github.com/satori/go.uuid/commit/0ef6afb2f6cdd6cdaeee3885a95099c63f18fc8c is still in his `master` branch and will still break folks using our repository in conjunction with `go get`.

Reply:
https://github.com/Azure/azure-sdk-for-go/issues/947 is related to this problem.

Reply:
I think he released a new version. we can take a dependency on that and use the new way  of doing things.

Reply:
But he has tagged a 1.2.0 release without breaking changes.
If we do `go get sdk` and `go get guid`, of course it would be broken, but truth is commits with breaking changes can happen anytime in any of our dependencies. `go get` will never protect us from breaking changes.
That is why the dep constraint is not against `master`, but `1.2.0`.

Reply:
@mcardosos so you are saying that release doesn't have the breaking changes or just that he upgraded only to minor version with the breaking change ?

Reply:
The upgrade to minor version does not have breaking changes. Master has breaking changes.

Reply:
I have talked offline with @marstr , the work done on this PR for uuid should be postponed.

Reply:
ok so you are saying that there is no release yet with the breaking change ? yes in that case we should postpone, but if he has let me know what would be the problem.

Reply:
There is no release yet with the breaking change.